alembic/templates/multidb/alembic.ini.mako

```diff
@@ -79,6 +79,12 @@ sqlalchemy.url = driver://user:pass@localhost/dbname2
 # black.entrypoint = black
 # black.options = -l 79 REVISION_SCRIPT_FILENAME
 
+# format using "ruff" - use the exec runner, execute a binary
```

Comment:
**mike bayer** (zzzeek) wrote:

same doc change as we agree on w/ the other file I noted

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/4788